webapplication/aoi/management/commands/_notebook.py

```diff
@@ -23,6 +23,14 @@
 
 THREAD_SLEEP = 10
 
+def clean_container_logs(logs):
+    log_lines = logs.split('\n')
+    log_lines = [re.sub(r'^\s*\d+\s*', '', line) for line in log_lines]
+
+    log_text = ''.join(log_lines)
+    log_text = re.sub(r'\x1b\[[0-9;]*m', '', log_text)
+    return log_text
```

Comment:
An example of what it does would be nice.

Reply:
fixed